### ISSUE TITLE
ebay-dialog: padding-top for dialog__main in taking up a lot of space in ds4

### ISSUE THREAD

**Issue by swathip88** [eBay/ebayui-core#1183]
# Bug Report

## Description
In ebayui-dialog, when there is `dialog__header` and `dialog__main` there is padding top for `dialog__main`. This is eating up a lot of space on the main part of the dialog

<img width="1428" alt="Screen Shot 2020-07-24 at 12 48 17 PM" src="https://user-images.githubusercontent.com/15898444/88444035-f6289300-cdcf-11ea-9edb-785d35cb0183.png">

**Comment by ianmcburnie** [comment]
Yes, I'm not sure why that padding-top is there. We are overhauling dialogs in v11, but can probably to a quick fix on this before then.
